test/integration/scheduler_perf/util.go

```diff
@@ -44,7 +44,7 @@ import (
 // Notes on rate limiter:
 //   - client rate limit is set to 5000.
 func mustSetupScheduler() (schedulerConfigFactory *factory.ConfigFactory, destroyFunc func()) {
-	framework.DeleteAllEtcdKeys()
+	// framework.DeleteAllEtcdKeys()
```

Comment:
Needed to disable for compilation. I considered deleting it but, clearly it was originally there for some purpose.  Once these are working again we can determine wether or not we want to re-enable etcd deletion. As of now, a new etc instance that would typically get started by the test wrapper makes this line not necessary unless running on a persistent instance